app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,22 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    # @param [Hash] options from blacklight invocation of helper_method
+    # @return the html_safe facet links separated by " > " to indicate facet hierarchy
```

Comment:
Add an example here of what you expect the solr document to look like, and the example configuration for using this method.